src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -265,22 +263,24 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
           <Layout.Body>
             <Layout.Main fullWidth>
               {!this.tryRenderOnboarding() && (
-                <StyledButtonBar merged active={status}>
-                  <Button
-                    to={{pathname, query: openIncidentsQuery}}
-                    barId="open"
-                    size="small"
-                  >
-                    {t('Active')}
-                  </Button>
-                  <Button
-                    to={{pathname, query: closedIncidentsQuery}}
-                    barId="closed"
-                    size="small"
-                  >
-                    {t('Resolved')}
-                  </Button>
-                </StyledButtonBar>
+                <NavWrapper>
```

Comment:
Is this needed? There is only one child

Reply:
Nope, this is leftover cruft from what I was fooling around with.